DisplaySettings/DisplaySettings.cpp

```diff
@@ -4588,7 +4578,7 @@ namespace WPEFramework {
 
 			std::vector<int> sad_list;
 			for (int i=0; i<shortAudioDescriptorList.Length(); i++) {
-                            LOGINFO("Short Audio Descriptor[%d]: %ld \n",i, shortAudioDescriptorList[i].Number());
+                            LOGINFO("Short Audio Descriptor[%d]: %d \n",i, (int) shortAudioDescriptorList[i].Number());
```

Comment:
What's the problem with printing int64_t so that you decided to cast  int64_t to int?

Reply:
Will fix it.